src/legacy/ui/public/styles/_mixins.scss

```diff
@@ -105,12 +105,12 @@
   @keyframes kibanaFullScreenGraphics_FadeIn {
     from {
       opacity: 0;
-      transform: translateY(200px), scale(.75);
+      transform: translateY(200px) scale(.75);
```

Comment:
dear @elastic/kibana-design , this CSS was invalid, so I've fixed it, and now the login screen fades in like this 

![image](https://user-images.githubusercontent.com/463851/87023912-b9a64600-c1d8-11ea-8aa5-ee60aac0e2e1.gif)
I think this can be removed? thx!

Reply:
I'd opt for removing the scaling. It doesn't look right when it includes the whole background layer.

Reply:
removed it. was btw the cause for failing functional tests 😄